mcd-sel-fj/scraped-data/[2022-08-11 01:07:46] mcd-sel-fj.csv

```diff
@@ -0,0 +1,41 @@
+,Date,Day,Territory,Menu Item,Price (FJD),Price (USD),Category,Menu
+1,2022/08/11,Thu,Fiji,Big Mac (S/R) -Beef,14.00,6.39,Classic McValue Meals,Regular
+2,2022/08/11,Thu,Fiji,Cheeseburger (S/R) - Beef,10.20,4.66,Classic McValue Meals,Regular
+3,2022/08/11,Thu,Fiji,Quarter Pounder (S/R) - Beef,14.45,6.60,Classic McValue Meals,Regular
+4,2022/08/11,Thu,Fiji,Filet-O-Fish (S/R),11.70,5.34,Classic McValue Meals,Regular
+5,2022/08/11,Thu,Fiji,McFeast (S/R) - Beef,15.80,7.22,Classic McValue Meals,Regular
```

Comment:
Category names are now populated in the appropriate Pascal Case.
We can re-enable the auto workflow now.